src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -284,9 +287,23 @@ def _resolve_default_value_from_config_file(self, arg, overrides):
         self._add_vscode_extension_metadata(arg, overrides)
 
         # same blunt mechanism like we handled id-parts, for create command, no name default
-        if self.name.split()[-1] == 'create' and overrides.settings.get('metavar', None) == 'NAME':
+        if not (self.name.split()[-1] == 'create' and overrides.settings.get('metavar', None) == 'NAME'):
+            super(AzCliCommand, self)._resolve_default_value_from_config_file(arg, overrides)
+
+        self._resolve_default_value_from_local_context(arg, overrides)
+
+    def _resolve_default_value_from_local_context(self, arg, overrides):
+        actions = overrides.settings.get('lc_actions', None)
```

Comment:
I'd prefer using full name `local_context_actions` for readability.

Reply:
let's gather more suggestion here. As each of us may need to add these attributes if the parameters need to support local context.

Want to here suggestion from other guys. Thanks